packages/contracts-bedrock/test/kontrol/proofs/OptimismPortal.k.sol

```diff
@@ -20,78 +20,35 @@ contract OptimismPortalKontrol is DeploymentSummary, KontrolUtils {
         superchainConfig = SuperchainConfig(superchainConfigProxyAddress);
     }
 
-    /// TODO: Replace struct parameters and workarounds with the appropriate
-    /// types once Kontrol supports symbolic `bytes` and `bytes[]`
-    /// Tracking issue: https://github.com/runtimeverification/kontrol/issues/272
+    /// @custom:kontrol-length-equals _withdrawalProof: 10,
+    /// @custom:kontrol-length-equals _withdrawalProof[]: 600,
```

Comment:
Got some data on the bounds here if it helps: https://dune.com/queries/3433954/5768623

Based on that, the min proof length is 5 and the max we've seen is 8, so we should bump that length to 9 or 10 to be conservative there. The element lengths range from 32 to 532 bytes, so using 32 and 600 as the bounds there would be great

Reply:
Given that, for the time being, we cannot enforce bounds on the elements of `bytes[]`, what would be your preferred approach? Set the concrete length to 600?

Reply:
Yea I think setting to element length to 600 bytes is good